### PR TITLE
Belen add logs to git checkout

### PR DESCRIPTION
**Pull request by belugarciafava**
We are having issues while trying to checkout the repository see: https://buildkite.com/canva-org/web-test-base/builds/337662#b59f36a3-4637-4248-98f3-524915b29371
Github support got back to us asking to add
```
GIT_TRACE=1 GIT_CURL_VERBOSE=1 git push -u origin master
```
to git operations to have more logging info

Not sure how to deploy the checkout plugin or add a new version of it

### CODE REVIEW

**Review by arromer — APPROVED** [review]

**Comment by arromer** [comment]
to release new version of the plugin we need to update all the occurrences of the old commit with the new commit once this is merged into master, in canva and infrastructure repos

**Review by uri-canva — CHANGES_REQUESTED** [review]
This is going to be extremely verbose, can you pipe it to a file, upload the file via `buildlkite-agent artifact upload` and then print out a message in the logs? Also double check what the performance impact of tracing is, we might not want to have it always enabled.

File: hooks/checkout [hunk 1]
```diff
@@ -101,9 +101,9 @@ checkout() {
     exit_code=0
     timeout "${GIT_REMOTE_TIMEOUT}" git fetch -v --no-tags origin "${BUILDKITE_BRANCH}" || exit_code=$?
```
Comment (arromer) [inline hooks/checkout:102]: maybe we should add it here too?

**Comment by danjuv** [comment]
Moving this to #41 so I can test this before merging into master

**Pull request closed without merging**